onko_dicom.py

```diff
@@ -85,10 +106,56 @@ def open_file(self):
             logger.info("open_file user canceled open operation")
             return
 
-        self.plot_w.plot_dcm(full_path[0])
-        self.close_action.setEnabled(True)
-
-        logger.info("open_file completed within OnkoDicom")
+        """Error handling"""
+        try:
+            logger.info("Attempting to graph/open file (%s)", full_path[0])
+            self.plot_w.plot_dcm(full_path[0])
+            self.close_action.setEnabled(True)
+            logger.info("successfully opened graph/file (%s)", full_path[0])
+
+        except pydicom.errors.InvalidDicomError as err:
+            logger.error("(%s): InvalidDicomError, Missing Dicom Header. Error:(%s)", full_path[0], err)
+            response = self.err_msg('Error', 'InvalidDicomError, Missing Dicom Header. \n\nError: '
+                                    + '<br>'.join([str(err)]))
+            if response == 'F':
+                self.plot_w.force_plot_dcm(full_path[0])
+                self.close_action.setEnabled(True)
+                logger.info("successfully force opened graph/file (%s)", full_path[0])
+            else:
+                pass
+
+        except AttributeError as err:
+            logger.error("(%s): AttributeError, Missing Attribute. Error:(%s)", full_path[0], err)
+            response = self.err_msg('Error', 'AttributeError, Missing Attribute. \n\nError: ' + '<br>'.join([str(err)]))
+            if response == 'F':
+                self.plot_w.force_plot_dcm(full_path[0])
+                self.close_action.setEnabled(True)
+                logger.info("successfully force opened graph/file (%s)", full_path[0])
+            else:
+                pass
+
+        except NotImplementedError as err:
+            try:
+                logger.error("(%s): NotImplementedError. Error:(%s)", full_path[0], err)
+                self.err_msg('Error', 'NotImplementedError. \n\nError: ' + ''.join([str(err)]))
+            except ValueError as err:
+                logger.error("(%s): ValueError. Error:(%s)", full_path[0], err)
+                response = self.err_msg('Error', 'NotImplementedError and ValueError. \n\nError: ' +
+                                        "Unable to decode pixel data  with a transfer syntax UID"
+                                        "as there are no pixel data handlers "
+                                        "available that support it. Please see the pydicom "
+                                        "documentation for information on supported transfer syntaxes\n\n"
```

Comment:
syntax's

Reply:
lol, This is the original text of the pydicom error

Reply:
oh haha thats weird